### ISSUE TITLE
MS SQL Server: connect works, basic query fails (RODBC works fine)

### ISSUE THREAD

**Issue by JockLawrie** [JuliaDB/ODBC.jl#85]
Hi there,

I am connecting to a remote SQL Server instance from a windows machine.
My Julia script (below) makes the connection, but fails when making a query - see the _Get data_ section in the script.
For reference, I have included below an almost identical R script that works.
I am running Julia version 0.3.8 on Windows, and ODBC.jl version 0.3.10.

Any ideas?

Cheers,
Jock

The Julia version:
```
### Init
using ODBC
con = ODBC.connect("my_dsn"; usr = "dsn_username", pwd = "dsn_password")

### Get data
# Both of these lines give the following error:
#     ERROR: [ODBC]: SQLExecDirect failed; Return Code: SQL_DEFAULT_ERROR in error at error.jl:21 (repeats 2 times)
tablenames = ODBC.query("SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE table_type = 'BASE TABLE'", con)
df1 = ODBC.query("select top 10 * from Event", con)

### Clean up
disconnect(con)
```

The R version:
```
### Init
library(RODBC)
con <-odbcConnect("my_dsn", uid="dsn_username", pwd="dsn_password")

### Get data
tablenames <- sqlQuery(con, "SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE table_type = 'BASE TABLE'")
df1 <- sqlQuery(con, "select top 10 * from Event")

### Clean up
close(con) 
```

**Comment by AndyGreenwell** [comment]
@JockLawrie The error message that you included above is the same as the error message seen in issue #86 that I filed.

Are you on a 64-bit platform executing 64-bit Julia against a 64-bit SQL Server?  If so, have you set a System DSN entry for your ODBC data source and could you try executing `const ODBC.odbc_dm = "C:\\Windows\\System32\\odbc32.dll" prior to executing `using ODBC`?

**Comment by JockLawrie** [comment]
@AndyGreenwell thanks for the suggestion.
I conducted a little further testing with and without your suggested  ```const odbc_dm = "C:\\Windows\\System32\\odbc32.dll"``` prior to ```using ODBC```. The results are the same though, and a little bit weird.

For example, if I repeat the query ```df1 = ODBC.query("select top 10 * from Event", con)``` several times I get any of the following 3 different responses seemingly randomly:
- ```ERROR: [ODBC]: SQLExecDirect failed; Return Code: SQL_DEFAULT_ERROR in error at error.jl:21 (repeats 2 times)```
- ```ERROR: invalid UTF-8 sequence in convert at utf8.jl:163 (repeats 2 times)```
- valid result (a data frame with the correct data in it)

Perhaps this is related to https://github.com/quinnj/ODBC.jl/issues/67?

**Comment by quinnj** [comment]
should be resolved by 7ae7ebfac60ed3ffb1ccefd552993a3073f8b77b; please re-open if you're still seeing issues
